### PR TITLE
Align table columns

### LINKED ISSUE #119: Listing page: Spacing between icons is too large
Screen Shot 2019-10-30 at 10 00 03 AM

vs 

Screen Shot 2019-10-30 at 12 52 00 PM

Introduced https://github.com/canonical-web-and-design/jaas-dashboard/pull/111

### LINKED ISSUE #140: Listing page: Alignments are off
Please see design for the right alignments, and the width of the first column:

dev:

### LINKED ISSUE #142: Listing page: Width of columns not consistent with design
The width of the columns needs to be adjusted per design, because of the different length of the content, we need to tweak the table:

### LINKED ISSUE #146: Details page: Width of the columns
Per design, the width of the columns needs to be tweaked according to the design and content

### LINKED ISSUE #179: Model detail page / Responsiveness breaking point for 15" laptop
Can we tweak the breaking point media queries for the responsive layout at 1580 instead 1680 in order to have the extended layout on 15" laptop screen?

At the moment in 1680 resolution I already see the smaller topology column.

### LINKED ISSUE #180: Model detail page / Responsiveness breaking point affects typography
I wouldn't change the size of the typography from 1680 breaking point:

> 1680:

< 1680:

I would keep the same size

CC @spencerbygraves @cristinadresch

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Align table columns
- Switch machine and unit icons in config column
- Capitalise td strings to match designs
- Turn off Vanilla text increases at larger breakpoints
- Tweak breakpoint for topology panel to 1580px

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View models and model details tables and verify that columns align as per the design

If you add this snippet to `index.scss`, you can see the column alignment visually.

```css
td {border-right: 1px solid orangered}
```

Fixes #119, 
Fixes #140, 
Fixes #142, 
Fixes #180, 
Fixes #146,
Fixes #179

### CODE REVIEW

**Comment by barrymcgee** [comment]
Screenshot 2019-11-06 at 16 03 49

**Comment by barrymcgee** [comment]
Screenshot 2019-11-06 at 16 04 07

**Comment by hatched** [comment]
QA Notes:
cc @spencerbygraves 
- [model list] The `configuration` and `cloud/region` feel quite tight when we have all that open space to the right of the region, thoughts on loosening that up a bit?
- [model details] While the columns might be aligned the headers are not making it look 'broken' on more sparse models when you can see all the headers on one screen.
Screen Shot 2019-11-06 at 11 27 43 AM
- [machine table] The machine number/ip are given a ton of extra space there. Maybe we give some space to the `status` and the rest to the `message` which we know already overflows. 
Screen Shot 2019-11-06 at 11 23 30 AM
- [relation table] There is considerable wrapping here can we adjust the spacing to avoid this as much as possible? Maybe have all columns dynamic width? 
Screen Shot 2019-11-06 at 11 24 38 AM

File: src/scss/custom/_status_icons.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
   display: inline-block;
   padding-left: 1.5rem;
   position: relative;
+  text-transform: capitalize;
```
Comment (hatched) [inline src/scss/custom/_status_icons.scss]: should the `.status-icon` have the new `u-capitalize` class added instead?

**Comment by hatched** [comment]
I need to land this branch as it conflicts with mine. Please address all follow-ups.

**Comment by hatched** [comment]
Issues have been created for the follow-ups.

**Pull request merged**